src/cli/tc.rs

```diff
@@ -54,51 +54,43 @@ pub fn run() -> Result<()> {
 }
 
 pub async fn init(file_path: &str) -> Result<()> {
-  let tailcallrc: resource::Resource<str> = resource_str!("examples/.tailcallrc.graphql");
+  let file_name = ".tailcallrc.graphql";
+  let file_name_yml = ".graphqlrc.yml";
```

Comment:
As per the issue, we should just drop the graphqlrc.yml file.

Reply:
the issue doesn't state that file should be removed. And it's mainly used for LSP integration that was initial intention for adding `init` command https://github.com/tailcallhq/tailcall/issues/402.

Could we revisit this? I think it shouldn't be removed.

Reply:
actually, I would even add additional file like `examples/jsonplaceholder.graphql` to make init command suitable for quick bootstrapping and testing. In that case after calling `tailcall init` we can quickly start the server with example configuration

Reply:
My bad. I meant:
1. We should NOT delete the `tailcallrc.graphql`. 
2. We should delete the `.graphql.yml`